phase1/gce/do

```diff
@@ -17,15 +24,16 @@ generate_token() {
 gen() {
   TOKEN=$(generate_token)
   mkdir -p ${TMP_DIR}
-  jsonnet -J ../../ --multi ${TMP_DIR} all.jsonnet
+  jsonnet -J ../../ --multi ${TMP_DIR} --tla-code-file cfg=${CONFIG_JSON_FILE} all.jsonnet
   echo "kubeadm_token = \"$TOKEN\"" > ${CLUSTER_NAME}/terraform.tfvars
 }
 
 deploy() {
   gen
   terraform apply -var-file=${CLUSTER_NAME}/terraform.tfvars -state=${CLUSTER_NAME}/terraform.tfstate ${TMP_DIR}
 
-  if [[ "${WAIT_FOR_KUBECONFIG:-}" == "y" ]]; then
+  PHASE2=$(jq -r '.phase2.provider' ${CONFIG_JSON_FILE})
```

Comment:
Yeah, after thinking more about it, I'm on board with this direction. If there are complaints, I think that the better behavior will be to allow opting-out of the functionality instead of requiring opting-in.